stdlib/regex.gr

```diff
@@ -387,7 +394,7 @@ enum ParsedRegularExpression {
   RELookbehind(ParsedRegularExpression, Bool, Box<Number>, Box<Number>, Number, Number), // regex, is-match, lb-min, lb-max, n-start, num-n (lb-xx values patched in later)
   RECut(ParsedRegularExpression, Number, Number, Bool), // regex, n-start, num-n, needs-backtrack
   REReference(Number, Bool), // n, case-sensitive
-  RERange(List<(Number, Number)>),
+  RERange(RERange),
```

Comment:
How is `RERange(RERange)` used? Do consumers need to have access to the list?

Reply:
Nah, they don't.